public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -252,12 +290,20 @@ export class Alerts extends Component<AlertsProps, AlertsState> {
     this.setState({ flyoutData: alertItem ? { alertItem } : undefined });
   }
 
+  // Function to show the tooltip
+  showTooltip = (errorMessage) => {
+    this.setState({ isTooltipVisible: true, errorMessage });
+  };
+
+  // Function to hide the tooltip
+  hideTooltip = () => {
+    this.setState({ isTooltipVisible: false });
+  };
   generateVisualizationSpec(alerts: AlertItem[]) {
     const visData = alerts.map((alert) => {
       const time = new Date(alert.start_time);
       time.setMilliseconds(0);
       time.setSeconds(0);
-
```

Comment:
Can we revert this change? We generally tend to keep an empty line before a return statement